tests/cupy_tests/statics_tests/test_meanvar.py

```diff
@@ -88,11 +88,24 @@ def test_external_mean_axis(self, xp, dtype):
         a = testing.shaped_arange((2, 3, 4), xp, dtype)
         return xp.mean(a, axis=1)
 
+    @testing.for_all_dtypes(no_complex=True)
     @testing.numpy_cupy_allclose()
-    def test_mean_all_dtype(self, xp):
-        a = xp.full((2, 3, 4), 123456789, dtype=numpy.int64)
+    def test_mean_real_dtypes_as_float(self, xp, dtype):
+        a = xp.full((2, 3, 4), 123456789, dtype=dtype)
         return xp.mean(a, dtype=numpy.float64)
```

Comment:
This is a test specifically for a bug fix #2076. It's better not to generalize it.
(I think the test name is inappropriate.)